.github/workflows/deploy.yml

```diff
@@ -104,6 +106,12 @@ jobs:
               --set-string config.nodes[2].keys.private=${{ steps.secrets.outputs.NODE_KEY_PRIVATE_VALIDATOR_1 }} \
               --set-string config.nodes[2].keys.public=${{ steps.secrets.outputs.NODE_KEY_PUBLIC_VALIDATOR_1 }} \
               --set config.nodes[2].sessionInjectionEnabled=false \
+              --set-string config.nodes[3].name=validator2 \
+              --set config.nodes[3].bootnode=true \
+              --set config.nodes[3].validator=true \
+              --set-string config.nodes[3].keys.private=${{ steps.secrets.outputs.NODE_KEY_PRIVATE_VALIDATOR_2 }} \
+              --set-string config.nodes[3].keys.public=${{ steps.secrets.outputs.NODE_KEY_PUBLIC_VALIDATOR_2 }} \
+              --set config.nodes[3].sessionInjectionEnabled=false \
```

Comment:
please remove this option `sessionInjectionEnabled`, removed on chart

Reply:
Done